GiniVision/Classes/ImageMetaInformationManager.swift

```diff
@@ -287,6 +288,13 @@ internal class ImageMetaInformationManager {
         return existingUUID ?? NSUUID().uuidString
     }
     
+    fileprivate func deviceOrientation() -> String {
```

Comment:
Normally, I'd say that a method about a device orientation shouldn't be in a class that deals with EXIF tags. However, I know that `ImageMetaDataManager` desperately needs to be re-written.

So no action required here, just wanted to point it out.

Reply:
Talking about this with Enrique we realized, that getting the current device orientation every time the user comment is updated can lead to false values: take pic in portrait, review it and rotate it in landscape -> device orientation will be changed to landscape, although the pic was taken in portrait. Thank you @nsobadjiev! 👍👍👍👍